sim/src/main/cc/fasedtests/test_harness_bridge.cc

```diff
@@ -24,6 +24,10 @@ test_harness_bridge_t::test_harness_bridge_t(
 // it then reads uarch event counts from the FASED instance and compares them against
 // expected values
 void test_harness_bridge_t::tick(){
+  // Wait for reset to complete.
+  if (sim->actual_tcycle() < 100)
```

Comment:
Why is this needed?

Reply:
There's no longer a blocking call to target_reset. This little hack does the job for these tests.